phys/module_microphysics_driver.F

```diff
@@ -569,6 +569,9 @@ smaw3d, &    !Normalized Galperin instability function for momentum  ( 0<= <=4.9
    REAL, DIMENSION(ims:ime,kms:kme,jms:jme), INTENT(INOUT) ::  & ! G. Thompson
                  re_cloud, re_ice, re_snow
    INTEGER, INTENT(IN):: has_reqc, has_reqi, has_reqs
+! new vars for LWP and PWAT diagnostics
```

Comment:
@thomasedds 
Thomas,
We don't need this line. 
What ever is used for spacing in the indexing, choose one.
Why is one new field optional, and the other is not?